module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -658,39 +673,22 @@ public function getHolding($bibId, array $patron = null, array $options = [])
      */
     protected function getDueDate($itemId, $showTime)
     {
-        $dueDateValue = '';
-        $query = [
-            'query' => 'itemId==' . $itemId
-        ];
+        $query = 'itemId==' . $itemId;
         foreach ($this->getPagedResults(
             'loans',
             '/circulation/loans',
-            $query
+            compact('query')
         ) as $loan) {
             // many loans are returned for an item, the one we want
             // is the one without a returnDate
-            if (!isset($loan->returnDate)) {
-                $dueDate = new DateTime(
-                    $loan->dueDate,
-                    new DateTimeZone('UTC')
-                );
-                $localTimezone = (new DateTime)->getTimezone();
-                $dueDate->setTimezone($localTimezone);
-                $dueDateValue = $this->dateConverter
-                    ->convertToDisplayDate(
-                        'U',
-                        $dueDate->format('U')
-                    );
-                if ($showTime) {
-                    $dueDateValue = $this->dateConverter
-                        ->convertToDisplayDateAndTime(
-                            'U',
-                            $dueDate->format('U')
-                        );
-                }
+            if (!isset($loan->returnDate) && isset($loan->dueDate)) {
+                $dueDate = $this->getDateTimeFromString($loan->dueDate);
+                $method = $showTime
```

Comment:
Used a variable method name instead of writing out two different calls with exactly the same parameters. This also prevents us from converting the date twice when `$showTime` is true (which was happening previously, but didn't need to).